### PR TITLE
Increment of font size for password and seed at new wallet dialogue

### PR DESCRIPTION
**Pull request by raulgobr**

### CODE REVIEW

**Comment by avahowell** [comment]
Linting failed, looks like there's some indentation issues

**Comment by avahowell** [comment]
Also, can you provide screenshots for this change?

File: plugins/Wallet/js/components/newwalletdialog.js [hunk 1]
```diff
@@ -7,13 +7,9 @@ const NewWalletDialog = ({password, seed, showConfirmationDialog, actions}) => {
 			{showConfirmationDialog ? <ConfirmationDialog /> : null}
 			<p> You have created a new wallet!  Please write down the seed and password in a safe place.  If you forget your password, you won't be able to access your wallet. </p>
 			<h2> Seed: </h2>
-			<span className="newwallet-seed">
-				{seed}
-			</span>
+				<div id="new-wallet-seed"> <strong> {seed} </strong> </div>
```
Comment (avahowell) [inline plugins/Wallet/js/components/newwalletdialog.js:10]: Linting is complaining about this line and the one after, just need to remove the one tab of spacing and the test pass (verified on my machine). Looked at the new styling and it looks good.

**Comment by DavidVorick** [comment]
@johnathanhowell @stefioan do you think it's worth cleaning this up and getting it merged?

**Pull request closed without merging**